src/squlearn/qnn/qnnc.py

```diff
@@ -179,15 +179,17 @@ def predict_proba(self, X: np.ndarray) -> np.ndarray:
 
         return pred
 
-    def partial_fit(self, X: np.ndarray, y: np.ndarray, weights: np.ndarray = None) -> None:
+    def partial_fit(self, X, y, weights: np.ndarray = None) -> None:
```

Comment:
Add doc strings from the fit function array-like or sparse matrix of shape (n_samples, n_features) etc